### PR TITLE
ADD yolov4 in models.yml

### PR DESCRIPTION
**Pull request by cabelo**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [x] There is reference to original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: samples/dnn/models.yml [hunk 1]
```diff
@@ -15,6 +15,30 @@ opencv_fd:
...
+  mean: [0, 0, 0]
+  scale: 0.00392
+  width: 416
+  height: 416
+  rgb: true
+  classes: "object_detection_classes_yolov3.txt"
+  sample: "object_detection"
+
+yolov4-tiny-voc:
+  model: " yolov4-tiny.weights"
```
Comment (alalek) [inline samples/dnn/models.yml:32]: Did you check your code locally?

**Review by alalek — CHANGES_REQUESTED** [review]

File: samples/dnn/models.yml (hunk 1 above)
Comment (cabelo) [reply samples/dnn/models.yml:32]: Yes, @alalek i did check, below evidence:

./object_detection --config=yolov4-tiny.cfg --classes=coco.names --model=yolov4-tiny.weights --width=416 --height=416 --scale=0.00392 --input=image-demo.jpg

Thank you for your concern

image

File: samples/dnn/models.yml [hunk 2]
```diff
@@ -15,6 +15,30 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO object detection family from Darknet (https://github.com/AlexeyAB/darknet)
+# Might be used for YOLOv4 and TinyYolov4
+yolov4:
```
Comment (dkurt) [inline samples/dnn/models.yml:20]: Again, there is already "yolo" alias. No need to duplicate at for every model.

File: samples/dnn/models.yml [hunk 3]
```diff
@@ -15,6 +15,30 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO object detection family from Darknet (https://github.com/AlexeyAB/darknet)
+# Might be used for YOLOv4 and TinyYolov4
```
Comment (dkurt) [inline samples/dnn/models.yml:19]: Please consider comments from previous PR

File: samples/dnn/models.yml (hunk 2 above)
Comment (cabelo) [reply samples/dnn/models.yml:20]: removed

**Review by dkurt — APPROVED** [review]
:+1:

File: samples/dnn/models.yml [hunk 4]
```diff
@@ -15,8 +15,10 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO4 object detection family from Darknet (https://github.com/AlexeyAB/darknet)
 # YOLO object detection family from Darknet (https://pjreddie.com/darknet/yolo/)
-# Might be used for all YOLOv2, TinyYolov2 and YOLOv3
+# Might be used for all YOLOv2, TinyYolov2, YOLOv3, YOLOv4 and TinyYolov4
+
 yolo:
```
Comment (alalek) [inline samples/dnn/models.yml:22]: Keep style consistent in this file and remove added extra line.

Comment (cabelo) [reply samples/dnn/models.yml:21]: done

**Review by alalek — APPROVED** [review]

**Pull request merged**